.github/workflows/release.yml

```diff
@@ -22,7 +23,8 @@ env:
 
 jobs:
   macos-release:
-    if:  startsWith(github.head_ref, 'release/')
+    # if the workflow was manually dispatched, or if someone commented on the PR with /release 
+    if: github.event_name == 'workflow_dispatch' || (startsWith(github.ref_name, 'release/') && github.event.issue.pull_request && startsWith(github.event.comment.body, '/release'))
```

Comment:
Do we even still need the `startsWith(github.ref_name, 'release/')` condition?  We can create a release branch, then manually trigger with a comment when desired.

This would also prevent us from having to stop the release workflow when a change is pushed to the release branch that is not intended to run the release workflow.

Reply:
I kept in the release branch restriction so that the `/release` comment will only work on release branch PRs. This adds a bit of safety from accidentally triggering the release workflow on other PRs.